api service integration changes (#5958)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/index.md

Reviewer: SCIM entitlement management properties section shouldn't be in the general index.md. It only applies to SCIM integrations, so it should be added into the SCIM snippets.

Reviewer: This isn't a common requirement, can you move this to the SCIM snippet?

#### <StackSnippet snippet="protocol-name" inline/> properties

> **Note:** The instructions on this page are for the **<StackSnippet snippet="protocol-fullname" inline/>** . <br>

> > If you want to change the instructions on this page, select the required option from the **Instructions for** dropdown list on the right.

Reviewer 2: ```suggestion
> If you want to change the instructions that yo see on this page, select a different option from the **Instructions for** dropdown list.
```
